docs/api/api.yml

```diff
@@ -0,0 +1,275 @@
+        accessRights:
+          $ref: "#/components/schemas/AccessRights"
+        license:
+          type: string
+          format: url
+
+
+        # For the data manager
+        messageForDataManager:
+          type: string
+        privacySensitiveDataPresent:
+          type: string
+          enum:
+            - yes
+            - no
+            - unspecified
+        acceptDepositAgreement:
+          type: boolean
+
+    # Components of DatasetMetadata below
+
+    AccessRights:
+      type: string
+      enum:
+        - OPEN_ACCESS
+        - REQUEST_PERMISSION
+    SchemedValue:
+      type: object
+      required:
+        - scheme
+        - value
+      properties:
+        scheme:
+          type: string
+        value:
+          type: string
+
+    Author:
+      type: object
+      properties:
+        titles:
+          type: string
+        initials:
+          type: string
+        insertions:
+          type: string
+        surname:
+          type: string
+        role:
+          $ref: "#/components/schemas/SchemedKeyValue"
+        ids:
+          type: array
+          items:
+            $ref: "#/components/schemas/SchemedValue"
+        organization:
+          type: string
+      oneOf:
+        - required:
+            - surname
+            - initials
+        - required:
+            - organization
+      example:
+        titles: Prof Dr
+        initials: A
+        insertions:
+        surname: Einstein
+        role:
+          - scheme: datacite:contributorType
+          - key: DataManager
+          - value: "Data Manager"
+        ids:
+          - scheme: DAI
+            value: "123456789x"
+          - scheme: ISNI
+            value: "ISNI:000000012281955X"
+        organization: University of Zurich
+
+    Date:
+      type: object
+      required:
+        - value
+        - qualifier
+      properties:
+        scheme:
+          type: string
+        value:
+          type: string
+        qualifier:
+          type: string
+          enum:
+            - dcterms:created
+            - dcterms:available
+            - dcterms:date
+            - dcterms:dateAccepted
+            - dcterms:dateCopyrighted
+            - dcterms:dateSubmitted
+            - dcterms:issued
+            - dcterms:modified
+            - dcterms:valid
+
+    SchemedKeyValue:
+      type: object
+      required:
+        - scheme
+        - key
+        - value
+      properties:
+        scheme:
+          type: string
+        value:
+          type: string
+        key:
+          type: string
+    # Note: it is not possible to provide examples in combination with a reference to this
+    # schema. If you need examples, you'll have to spell out the complete type.
+    StringArray:
+      type: array
+      items:
+        type: string
+    IncludeSampleHeader:
+      type: boolean
+      description: Whether the resulting Agreement is a sample, or a final Agreement
```

Comment:
Is this called a "sample"? That sounds like "specimen" with some random data. I think "draft" would be a better term here.

Reply:
i agree, but this is what it is called. i merely described the status quo